src/components/Buttons/Button.test.js

```diff
@@ -0,0 +1,33 @@
+import { render } from '@testing-library/react'
+import Button from './Button'
+import user from '@testing-library/user-event'
+
+describe('Button', () => {
+  it('renders correctly', () => {
+    const { container, rerender } = render(
+      <Button disabled={false} handleClick={() => {}}>
+        Save this song
+      </Button>
+    )
+    expect(container.firstChild).toMatchSnapshot()
+
+    rerender(
+      <Button disabled={true} handleClick={() => {}}>
+        Save this song
+      </Button>
+    )
+    expect(container.firstChild).toMatchSnapshot()
```

Comment:
Maybe separate render test into two tests - one for disabled and one for enabled. Test could fail for two reasons.

Reply:
I think in this case there is no need to split the test as it is a simple snapshot test.